packages/react-components/lib/banner-tab.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { createStyles, makeStyles } from '@material-ui/core';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Badge from '@material-ui/core/Badge';
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    logo: {
+      maxWidth: 120,
+      opacity: 1,
+    },
+  }),
+);
+
+interface BannerTabProps {
+  logoPath?: string;
+}
+
+export const BannerTab = (props: BannerTabProps): JSX.Element => {
+  const { logoPath } = props;
+  const classes = useStyles();
+
+  return (
+    <Tabs
```

Comment:
In the previous design, the banner is one of the tabs in the navigation bar, but now in this it's a separate tab bar in itself. In that case, it makes sense to change this to a image or button (or whatever material's best component for this use case). While you are at it, also rename this to avoid confusion.

This simplifies stuff as you no longer have to deal with all the complexity of a tab.